### PR TITLE
Change WiX template to deploy powershell wrapper modules.

### PR DESCRIPTION
**Pull request by ksubrama**
This needs https://github.com/chef/chef/pull/3339 before it will work.

### CODE REVIEW

**Comment by schisamo** [comment]
Will this work on older versions of Windows?

**Comment by jdmundrawala** [comment]
I think this should be it's own feature. Opt-in sort of thing

**Comment by ksubrama** [comment]
@jdmundrawala I can make it a separate feature.

@schisamo If the current powershell available doesn't support module autoloading, then the installed files are harmless.  We export an extra environment variable but that's it.

**Comment by ksubrama** [comment]
@jdmundrawala It's a separate feature now.

**Comment by jdmundrawala** [comment]
:+1:

**Comment by jdmundrawala** [comment]
Probably will need to do the same thing for chefdk

**Comment by ksubrama** [comment]
@jdm This is ready for another look.

**Comment by jdmundrawala** [comment]
You pinged the wrong person

File: resources/chefdk/msi/source.wxs.erb [hunk 1]
```diff
@@ -32,15 +32,27 @@
...
           <Directory Id="PROJECTLOCATION" Name="chefdk" >
+            <Directory Id="PROJECTLOCATIONBIN" Name="bin" >
+              <Component Id="ChefDkPath" Guid="{AEA5727E-DAD2-48CC-ADB8-38DD0EB46C62}" >
+                <Environment Id="Environment"
+                             Name="PATH" Action="set" Part="last" System="yes" Value="[PROJECTLOCATIONBIN]" />
+              </Component>
+            </Directory>
+            <Directory Id="PSMODULES" Name="modules" >
+              <Directory Id="PSMODULESCHEF" Name="chef" >
+                <Component Id="ChefPSModuleFile" Guid="{DF0E99BF-1877-4099-85A4-067C84A743F2}" >
```
Comment (jdmundrawala) [inline resources/chefdk/msi/source.wxs.erb]: I don't think we should be reusing guids

**Comment by jdmundrawala** [comment]
make sure to run this through the build and make sure it does what you want.

**Comment by ksubrama** [comment]
I made a local chef-client build and it seems to deploy things to the right place.  Module auto-loading seems to not be triggering though - but if I manually import the module (Import-Module chef), it does work.  I think this might be a limitation in powershell when a command of the same name already exists in the path.  Asking folks to add Import-Module chef to their ps profiles seems good enough to me for now.  If nobody complains, we can auto-add it in the future.

**Comment by ksubrama** [comment]
My local chefdk builds seem to work as well!  :ship:

**Pull request merged**